comparisons.py

```diff
@@ -0,0 +1,63 @@
+import matplotlib.pyplot as plt
```

Comment:
Looks like this is meant to be a one-off script, but I recommend renaming it something more descriptive.

Reply:
Yeah, I was thinking to just move it to the plotting notebook. Would that work?